landscape/client/manager/ubuntuproinfo.py

```diff
@@ -30,12 +33,45 @@ def get_data(self):
                           sort_keys=True)
 
 
+q = Queue()
```

Comment:
Having this be a global has me a tiny bit concerned. What if things start to pile up in it?

I think we should create a new `Queue` every time we would call `uastatus`, to be excessively safe.